### ISSUE TITLE
Autofix breaks comments removing zero/final line

### ISSUE THREAD

**Issue by Dimava** [gajus/eslint-plugin-jsdoc#1040]
## Expected behavior

When removing first or last line of the comment by autofixing, the `/*` / `*/` should remain

## Actual behavior

Opening and closing of the comment can be removed by autofixer

## ESLint Config

```js
{
    plugins: ['jsdoc'],
    extends: ['plugin:jsdoc/recommended-typescript'],
    rules: {
      'jsdoc/multiline-blocks': ['warn', {
        noZeroLineText: false,
        noFinalLineText: false,
      }],
}
```

## ESLint sample

The `@typedef` and `@module` lines get removed with comment open/close, completely breaking the code
```js
/** @typedef {Object} MyObject
 * @property {string} id - my id
 */

/** some text
  @module */
function f() { }
```

## Environment

- Node version: v18.12.1
- ESLint version 8.36.0
- `eslint-plugin-jsdoc` version: 43.0.7

**Comment by brettz9** [comment]
If you run eslint at the command line, which rule is reported as the cause of the error?

**Comment by brettz9** [comment]
Nevermind, looks like it's `check-tag-names`
